airbus/common/stac_utils.py

```diff
@@ -292,10 +454,26 @@ def update_stac_item_success(
 ):
     """Update the STAC item with the assets and success order status"""
     # Add all files in the directory as assets to the STAC item
-    for root, _, files in os.walk(directory):
-        for asset in files:
+    name_counter = {}
+    for root, dirs, files in os.walk(directory):
+        dirs.sort()
+        for asset in sorted(files):
             asset_path = os.path.join(root, asset)
-            asset_name = os.path.basename(asset_path)
+            asset_name, description = get_asset_details(asset_path, collection_id)
+
+            # Append row and column indexes to the asset name if they exist
+            match = re.search(r"_R\d+C\d+\.", asset_path.upper())
+            if match:
+                # Remove trailing dot originating from the extension
+                asset_name = asset_name + match.group(0)[:-1]
```

Comment:
You might be able to do something with capture group to avoid removing the dot at the end. I haven't tested this but maybe something like 

```suggestion
            match = re.search(r"(_R\d+C\d+)\.", asset_path.upper())
            if match:
                # Remove trailing dot originating from the extension
                asset_name = asset_name + match.group(1)
```

Reply:
No need to update this though

Reply:
Clean suggestion, tested in console and done